backend/server/server.go

```diff
@@ -45,6 +45,7 @@ func Start() {
 
 	// handle public user stuff
 	router.HandleFunc("/v0/userinfo/{username}", usershandlers.GetPublicUserResponse).Methods("GET")
+	router.HandleFunc("/v0/usercredit/{username}", usershandlers.GetUserCreditResponse).Methods("GET")
```

Comment:
if you do end up changing the handler function name, just remember to update it here as well.